docs/readmes/howtos/package.md

```diff
@@ -8,7 +8,7 @@ hide_title: true
 
 ## TL;DR
 
-1. Run `fab test package` on the host to create packages inside the
+1. Run `fab release package` on the host to create packages inside the
```

Comment:
can we add info on both fab commands?

Reply:
This doc already has a section called "creating a development package", so I'll leave this section alone.